miso-web/src/main/webapp/scripts/hot_index.js

```diff
@@ -0,0 +1,66 @@
+HotTarget.index = {
+  getCreateUrl: function() {
+    return Urls.rest.indices.create;
+  },
+  getUpdateUrl: function(id) {
+    return Urls.rest.indices.update(id);
+  },
+  requestConfiguration: function(config, callback) {
+    callback(config)
+  },
+  fixUp: function(index, errorHandler) {
+    if (index.family.fakeSequence) {
+      index.sequence = index.name;
+    }
+  },
+  createColumns: function(config, create, data) {
+    return [{
+      header: 'Family',
+      data: 'familyName',
+      readOnly: true,
+      include: true,
+      unpack: function(index, flat, setCellMeta) {
+        flat.familyName = index.family ? index.family.name : config.indexFamily.name;
+      },
+      pack: function(index, flat, errorHandler) {
+        if (!index.family) {
+          index.family = config.indexFamily;
+        }
+      }
+    }, HotUtils.makeColumnForInt("Position", true, 'position', HotUtils.validator.integer(true, 1, 2)),
+        HotUtils.makeColumnForText('Name', true, 'name', {
+          validator: HotUtils.validator.requiredTextNoSpecialChars
+        }), HotUtils.makeColumnForText('Sequence', !config.indexFamily.fakeSequence, 'sequence', {
+          validator: HotUtils.validator.requiredTextNoSpecialChars
+        }), {
+          header: 'Sequences',
+          data: 'realSequences',
+          description: 'Values should be comma-separated. e.g. "ACACAC, GTGTGT"',
```

Comment:
Does this description get displayed anywhere?

Reply:
Column descriptions are added to the quick help section at the top of the page

Reply:
I'm not seeing them at the top of the Create Indices page, can you look into why that might be?

Reply:
Are you working with a "fake sequence"/multi-sequence family? It's showing up for me

Reply:
Ah, that was the issue. Better now :+1: